src/ck-ldb/LBDatabase.C

```diff
@@ -867,19 +293,6 @@ int LBDatabase::Migrate(LDObjHandle h, int dest)
   return 1;
 }
 
-void LBDatabase::Migrated(LDObjHandle h, int waitBarrier)
-{
-  // Object migrated, inform load balancers
-
-  // subtle: callback may change (on) when switching LBs
-  // call in reverse order
-  //for(int i=0; i < migrateCBList.length(); i++) {
-  for(int i = migrateCBList.size()-1; i >= 0; i--) {
-    MigrateCB* cb = (MigrateCB*)migrateCBList[i];
-    if (cb && cb->on) (cb->fn)(cb->data, h, waitBarrier);
-  }
-}
-
 void LBDatabase::MetaLBResumeWaitingChares(int lb_ideal_period) {
```

Comment:
Should be part of LBDatabase or LBManager?

Reply:
It'll stay in LBDatabase for now. I think later on we should do a revamp of the Metabalancer stuff, it's too intertwined into stuff right now in my opinion.